[1;35m[WARNING]: While constructing a mapping from[0m
[1;35m/home/s/projects/raspicam/remotecam/files/setup/ansible/rpi_custom_user.yml,[0m
[1;35mline 2, column 3, found a duplicate dict key (gather_facts). Using last defined[0m
[1;35mvalue only.[0m

PLAY [Prepare keys for root (as 'pi', connect with password)] ******************

TASK [Set authorized_key for root user] ****************************************
[0;32mok: [192.168.1.30][0m

PLAY [Rename user (as 'root', connect with key)] *******************************

TASK [Create the 'zanzocam-rpi' group] *****************************************
[0;32mok: [192.168.1.30][0m

TASK [Create the 'zanzocam-rpi' user] ******************************************
[1;35m[WARNING]: The input password appears not to have been hashed. The 'password'[0m
[1;35margument must be encrypted for this module to work properly.[0m
